Sources/Lexical.swift

```diff
@@ -26,6 +26,23 @@ public func letter(_ source: [Character]) -> (Character, [Character])? {
   }
 }
 
+public func word(_ value: String) -> ([Character]) -> (String, [Character])? {
+  let wordParser = letter+ ^^ { (cs: [Character]) -> String in
+    print(cs)
```

Comment:
this should be removed and the lambda can be converted to single expression syntax